package.json

```diff
@@ -22,48 +23,34 @@
     "test": "test"
   },
   "scripts": {
-    "pretsc": "mkdir -p build && rm -rf build/*/** && cp package.json build/",
-    "prebuild": "mkdir -p build/site build/test-gallery",
+    "prebuild": "mkdir -p build/src",
     "build": "npm run build:only",
-    "build:only": "tsc && cp package.json build && browserify src/index.ts -p tsify -d -s vl | exorcist build/vega-lite.js.map > build/vega-lite.js",
-    "postbuild": "node node_modules/uglify-js/bin/uglifyjs build/vega-lite.js -cm --source-map content=build/vega-lite.js.map,filename=build/vega-lite.min.js.map -o build/vega-lite.min.js && npm run schema",
+    "build:only": "tsc && cp package.json build/src/ && rollup -c",
+    "postbuild": "uglifyjs build/vega-lite.js -cm --source-map content=build/vega-lite.js.map,filename=build/vega-lite.min.js.map -o build/vega-lite.min.js && npm run schema",
     "build:examples": "npm run data && TZ=America/Los_Angeles scripts/build-examples.sh",
     "build:examples-full": "TZ=America/Los_Angeles scripts/build-examples.sh 1",
     "build:example": "TZ=America/Los_Angeles scripts/build-example.sh",
     "build:toc": "bundle exec jekyll build -q && scripts/generate-toc",
-    "build:site": "browserify site/static/main.ts -p [tsify -p site] -d | exorcist build/site/main.js.map > build/site/main.js",
+    "build:site": "tsc -p site && webpack --config site/webpack.config.js",
     "build:versions": "scripts/update-version.sh",
-    "build:test-gallery": "browserify test-gallery/main.ts -p [tsify -p test-gallery] -d > build/test-gallery/main.js",
     "check:examples": "scripts/check-examples.sh",
     "check:schema": "scripts/check-schema.sh",
-    "clean": "rm -rf build && rm -f vega-lite.* & find -E src test site examples -regex '.*\\.(js|js.map|d.ts)' -delete && rm -f examples/compiled/*.png && find site/examples ! -name 'index.md' -type f -exec rm -f {} +",
+    "clean": "rm -rf build && rm -f examples/compiled/*.png && find site/examples ! -name 'index.md' -type f -delete",
```

Comment:
Why are there `.png` included?

Reply:
Because we need to have the geo examples as PNGs.